iocore/net/QUICNetVConnection.cc

```diff
@@ -1203,7 +1203,7 @@ QUICNetVConnection::_store_frame(ats_unique_buf &buf, size_t &offset, uint64_t &
     QUICConDebug("[TX] %s", msg);
   }
 
-  frames.push_back(std::move(frame));
+  frames.push_back({frame->id(), frame->generated_by()});
```

Comment:
Here is the change.

Reply:
Why not
```
frames.emplace_back(std::move(frame));
```
This would make it less likely that chances to the QUICFramePtr type would break this function.

Reply:
I'm not sure what you mean by "break this function", but I just couldn't come up with that.

Reply:
By "break this function" I mean make it necessary to change this function.